src/workloads/scale/InsertBigDocs.yml

```diff
@@ -0,0 +1,21 @@
+# This workload loads 500 rows into a collection, each of which is approximately 15MB in size.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/replication"
+
+Actors:
+
+- Name: Loader
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &DB test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 500
+    BatchSize: 1
+    Document:
+      x: {^RandomInt: {min: 0, max: 2147483647}}
+      string0: {^RandomString: {length: 15000000}}
```

Comment:
1. I would suggest using `string0: {^FastRandomString: {length: 15000000}}`. The tradeoff is *slightly* less entropy but much faster generation of the strings. I made this change on my laptop and genny was able to produce writes of around 80mb/sec versus only 25 mb/sec with regular `RandomString`.

2. Consider dividing up the load between a number of threads. I.e. change `Threads: 1` to something like `Threads: 10` and then divide `DocumentCount:` by the new threadcount to maintain the same number of documents. This will of course change what the workload is doing - doing multiple writes in parallel so may not be what you're aiming for. But since this is single-threaded as it is you're going to be waiting between each `insert_one` invocation for genny to produce the next document.

Reply:
1. Yeah good call. FastRandomString is perfect for this use case
2. I'm pretty sure we want just 1 thread for this particular workload. The 10 thread case may be interesting but can be done as future work (in the form of another phase)